src/coreclr/debug/createdump/crashinfo.cpp

```diff
@@ -219,6 +219,27 @@ CrashInfo::GatherCrashInfo(MINIDUMP_TYPE minidumpType)
     return true;
 }
 
+static const char*
+GetHResultString(HRESULT hr)
+{
+    switch (hr)
+    {
+        case E_FAIL:
+            return "The operation has failed";
+        case E_INVALIDARG:
+            return "Invalid argument";
+        case E_OUTOFMEMORY:
+            return "Out of memory";
+        case CORDBG_E_UNCOMPATIBLE_PLATFORMS:
+            return "The operation failed because debuggee and debugger are on incompatible platforms";
+        case CORDBG_E_MISSING_DEBUGGER_EXPORTS:
+            return "The debuggee memory space does not have the expected debugging export table";
+        case CORDBG_E_UNSUPPORTED:
+            return "The specified action is unsupported by this version of the runtime";
+    }
+    return "";
```

Comment:
nit
```suggestion
        case CORDBG_E_UNSUPPORTED:
            return "The specified action is unsupported by this version of the runtime";
        default:
            return "";
    }
```

Reply:
Since this is a nit and no other changes are needed (see comments on your other feedback), I'm going to skip this one.